packages/statusbar-extension/src/index.ts

```diff
@@ -59,6 +70,41 @@ const statusBar: JupyterFrontEndPlugin<IStatusBar> = {
       });
     }
 
+    const category: string = 'Main Area';
+    const command: string = 'toggle-jp-main-statusbar';
```

Comment:
Our convention for command IDs has been to use something like `'plugin:action'`, so can we make this  `'statusbar:toggle'`?